<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="denfense1">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>0</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="denfense2">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>170</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="data1">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>70</y>
      <width>70</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>攻击力：50
等级</string>
    </property>
   </widget>
   <widget class="QLabel" name="data2">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>240</y>
      <width>70</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>攻击力：
等级：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reStart">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>500</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="defense3">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>340</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="data3">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>430</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="update">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>500</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>升级</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
